airflow/sensors/sql.py

```diff
@@ -90,7 +105,7 @@ def _get_hook(self):
                 f"Connection type ({conn.conn_type}) is not supported by SqlSensor. "
                 + f"Supported connection types: {list(allowed_conn_type)}"
             )
-        return conn.get_hook()
+        return conn.get_hook(**self.hook_kwargs)
```

Comment:
… or this should be `conn.get_hook(self.hook_kwargs)`?

Reply:
I changed my mind about who should receive the destructured dict. I suppose you are right. because then we don't risk passing an argument to `get_hook` that is later popped out of the args, or unintendedly defined and taken out of the `get_hook` kwargs. I'll switch it the other way around. Just like discussed in the issue